test/puppetlabs/puppetdb/integration/smoke.clj

```diff
@@ -0,0 +1,29 @@
+(ns puppetlabs.puppetdb.integration.smoke
+  (:require [clojure.test :refer :all]
+            [puppetlabs.puppetdb.integration.fixtures :as int]))
+
+(deftest ^:integration simple-agent-run
+  (with-open [pg (int/setup-postgres)
+              pdb (int/run-puppetdb pg {})
+              ps (int/run-puppet-server [pdb] {})]
+    (testing "Agent run succeeds"
+      (let [{:keys [out]} (int/run-puppet-as "my-agent" ps pdb "notify { 'hello, world!': }")]
```

Comment:
`run-puppet` (well `run-bundle-exec` under it) will print all of that if the command fails